cns/restserver/util.go

```diff
@@ -236,37 +237,60 @@ func (service *HTTPRestService) updateIpConfigsStateUntransacted(req cns.CreateN
 	}
 
 	// Add the newIpConfigs, ignore if ip state is already in the map
-	service.addIPConfigStateUntransacted(req.NetworkContainerid, newIPConfigs)
+	service.addIPConfigStateUntransacted(req.NCVersion, req.NetworkContainerid, newIPConfigs)
```

Comment:
Can you add some logs to updateIpConfigsStateUntransacted function, it is missing logs overall. Specially when we are updating the state or if this api is returning error

Reply:
Sure.